torch/testing/_internal/common_methods_invocations.py

```diff
@@ -10222,7 +10222,11 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
            supports_forward_ad=True,
            supports_fwgrad_bwgrad=True,
            sample_inputs_func=sample_inputs_masked_select,
-           error_inputs_func=error_inputs_masked_select),
+           error_inputs_func=error_inputs_masked_select,
+           skips=(
+               # Compiler issue on ROCm. Might need to skip until ROCm5.5
```

Comment:
@jithunnair-amd All of these skips should have been applied only for the ROCm platform. Please look at how other tests are skipped (`active_if=TEST_WITH_ROCM`).

Reply:
Thanks @IvanYashchuk, my fault. I have added the ROCm-only skip logic.